test/lib/s3/select_object_contents/stream_test.exs

```diff
@@ -0,0 +1,35 @@
+defmodule ExAws.S3.SelectObjectContent.StreamTest do
+  use ExUnit.Case, async: true
+
+  import Support.BypassHelpers
+  alias ExAws.S3
+
+  describe "integration test" do
+    setup [:start_bypass]
+
+    test "stream SelectObjectContent results", %{bypass: bypass} do
```

Comment:
https://github.com/ex-aws/ex_aws/pull/1012 has to be merged for this test to pass